p751toolbox/print.go

```diff
@@ -0,0 +1,55 @@
+package p751toolbox
+
+import (
+	"reflect"
```

Comment:
You don't need reflect here. Please avoid it at all costs.

Reply:
reflect package dependency removed.